ovmf-vars-generator

```diff
@@ -166,6 +166,8 @@ def parse_args():
                         default='/usr/share/edk2/ovmf/OVMF_VARS.fd')
     parser.add_argument('--uefi-shell-iso', help='Path to uefi shell iso',
                         default='/usr/share/edk2/ovmf/UefiShell.iso')
+    parser.add_argument('--skip-testing', help='Skip testing generated vars',
```

Comment:
Nit: s/generated vars/generated "VARS" file/
(Otherwise it sounds like multiple files.)